src/devices/sowilo.ts

```diff
@@ -0,0 +1,17 @@
+import {Definition} from '../lib/types';
+import * as ota from '../lib/ota';
+import extend from '../lib/extend';
+
+const definition: Definition[] = [
+    {
+        zigbeeModel: ['L258'],
+        model: 'L258',
+        vendor: 'Sowilo DS',
+        description: 'Heimdall Pro. 5 channel RGBWW controller.',
+        extend: extend.light_onoff_brightness_colortemp_color({supportsHueAndSaturation: true, colorTempRange: [150, 575]}),
+        ota: ota.zigbeeOTA,
```

Comment:
I don't think we have an ota available for this device so please remove it.

Reply:
what's the procedure for OTAs ? I'm gonna remove this for now, but I can publish OTA files for this device.

Reply:
Ah great, you can add it here: https://github.com/Koenkk/zigbee-OTA